comments.php

```diff
@@ -72,7 +72,7 @@
 
 	<?php
 		// If comments are closed and there are comments, let's leave a little note, shall we?
-		if ( ! comments_open() && '0' != get_comments_number() && post_type_supports( get_post_type(), 'comments' ) ) :
+		if ( ! comments_open() && get_comments_number() && post_type_supports( get_post_type(), 'comments' ) ) :
 	?>
```

Comment:
I think I started doing that in default themes at some point. We don't really need a strict comparison here, we just need to know if it's truthy.